src/core/src/service.rs

```diff
@@ -61,7 +127,6 @@ impl Poet2Service {
     }
 
     pub fn get_block(&mut self, block_id: BlockId) -> Block {
-        debug!("Getting block {:?}", block_id);

```

Comment:
Why should this log be removed?

Reply:
The block_id's value is unknown as it's not holding any value,so it should be removed.